docs/training/hyperparameter/search-methods/hp-custom.rst

```diff
@@ -0,0 +1,177 @@
+.. _topic-guides_hp-tuning-det_custom:
+
+#######################
+ Custom Search Methods
+#######################
+
++----------------------------------------------------------------+
+| API reference                                                  |
++================================================================+
+| :doc:`/reference/reference-searcher/custom-searcher-reference` |
++----------------------------------------------------------------+
+
+Determined supports defining your own hyperparameter search algorithms and provides search runner
+utilities for executing them.
+
+Remember that a Determined experiment is a set of trials, each corresponding to a point in the
+hyperparameter space.
+
+To implement a custom hyperparameter tuning algorithm, subclass
```

Comment:
How about: 

To implement a custom hyperparameter tuning algorithm, subclass
:class:`~determined.searcher.SearchMethod`, overriding its event handler methods.
Note that overriding :meth:`~determined.searcher.SearchMethod.save_method_state` and
:meth:`~determined.searcher.SearchMethod.load_method_state` is not mandatory,
however, it is **recommended** as it enables fault tolerance.

To run the custom HP tuning algorithm, you can use:

- :class:`~determined.searcher.LocalSearchRunner` that executes on your local machine,
- :class:`~determined.searcher.RemoteSearchRunner` that runs on a Determined cluster.

SearchRunner executes the custom HP tuning algorithm and starts a multi-trial experiment
on a Determined cluster.